docs/executor-custom.md

```diff
@@ -101,11 +162,209 @@ echo "http://google.pl" | kubectl testkube tests create --name example-google-te
+- [Cypress runner implementation](https://github.com/kubeshop/testkube-executor-cypress/blob/main/pkg/runner/cypress.go)
+- [Curl runner implementation](https://github.com/kubeshop/testkube-executor-curl/blob/main/pkg/runner/runner.go)
+
+
+# Creating executor in other programming language (than `go`)
+
+([You can find full commented code example here](https://github.com/kubeshop/testkube-executor-example-nodejs/blob/main/app.js))
+
+For go-based executors we've prepared a lot of handy functions (like printing valid outputs or wrappers around calling external processes)
+In other languages (for now) you'll need to manage this by your own. 
+
+One thing which Testkube simplified is test content management. As we're supporting several different test content types (like string,uri,git-file,git-dir)
+The whole complexity of checking out or downloading is covered by Testkube. 
+
+Testkube will store it's files and directories in directory defined by `RUNNER_DATADIR` env 
+And will save `test-content` file for:
+- string content (e.g. postman collection is passed as string content read from json file)
+- uri (testkube will get content of file defined by uri)
+In case of git related content: 
+- testkube will checkout repo content in that directory
+
+We've created simple NodeJS executor (sorry for our Node skills we've tried the best ;) ) 
+
+Executor will get URI and try to call HTTP GET method on passed value, and will return 
+- success - when status code will be 200 
+- failed - otherwise 
+
+```javascript
+"use strict";
+
+const https = require("https");
+const fs = require("fs");
+const path = require("path");
+
+const args = process.argv.slice(2);
+if (args.length == 0) {
+  error("Please pass arguments");
+  process.exit(1);
+}
+
+var uri;
+if (!process.env.RUNNER_DATADIR) {
+  error("No valid data directory detected");
+  process.exit(1);
+}
+
+const testContentPath = path.join(process.env.RUNNER_DATADIR, "test-content");
+uri = fs.readFileSync(testContentPath, { encoding: "utf8", flag: "r"});
+
+https.get(uri, (res) => {
+    if (res.statusCode == 200) {
+      successResult("Got valid status code: 200 OK");
+    } else {
+      errorResult("Got invalid status code");
+    }
+  })
+  .on("error", (err) => { error("Error: " + err.message); });
+
+
+function errorResult(message) {
+  console.log(JSON.stringify({ "type": "result", "result": { "status": "error", "errorMessage": message, }}));
+}
+
+function successResult(output) {
+  console.log(JSON.stringify({ "type": "result", "result": { "status": "success", "output": output, }}));
+}
+
+// error will return error info not related to test itself (some issues with executor)
+function error(message) {
+  console.log(JSON.stringify({ "type": "error", "content": message, })); 
+}
+  
+```
+
+Code is ready and working - we're assuming defaults so `RUNNER_DATADIR` will be `/data` and our file will be in `/data/test-content` file.
```

Comment:
The code is ready and working. With the defaults assumed, `RUNNER_DATADIR` will be `/data` and the file will be saved in the `/data/test-content` directory.